packages/fuels/tests/providers.rs

```diff
@@ -308,46 +286,19 @@ async fn contract_deployment_respects_maturity() -> Result<()> {
             LoadConfiguration::default(),
         )
         .map(|loaded_contract| {
-            loaded_contract.deploy(wallet, TxParameters::default().with_maturity(maturity))
+            loaded_contract.deploy(wallet, TxPolicies::default().with_maturity(maturity))
         })
     };
 
-    let err = deploy_w_maturity(1u32)?.await.expect_err("Should not have been able to deploy the contract since the block height (0) is less than the requested maturity (1)");
-    assert!(matches!(
-        err,
-        Error::ValidationError(fuel_tx::CheckError::TransactionMaturity)
-    ));
+    let err = deploy_w_maturity(1u32)?.await.expect_err(
+        "Should not deploy contract since block height (0) is less than the requested maturity (1)",
+    );
+    assert!(matches!(err, Error::ValidationError(s) if s == "TransactionMaturity"));
 
     provider.produce_blocks(1, None).await?;
     deploy_w_maturity(1u32)?
         .await
-        .expect("Should be able to deploy now since maturity (1) is <= than the block height (1)");
-    Ok(())
-}
-
-#[tokio::test]
```

Comment:
Why are we deleting this test?

Reply:
Because this is not the case anymore. the `tx.script_gas_limit` should not match `max_gas_per_tx`